Formula/sql-translator.rb

```diff
@@ -110,7 +110,7 @@ def install
     # Disable dynamic selection of perl which may cause segfault when an
     # incompatible perl is picked up.
     # https://github.com/Homebrew/homebrew-core/issues/4936
-    bin.find { |f| rewrite_shebang detected_perl_shebang, f }
+    rewrite_shebang detected_perl_shebang, *bin.children
```

Comment:
You only changed one 😅

Reply:
ha oops, didn't save the rest of the files 💩

Reply:
This is exactly what I was imagining might have needed to be done.  Looks good.  Thanks!